src/10_functions.py

```diff
@@ -2,6 +2,9 @@
 
 # YOUR CODE HERE
 
+def is_even(n):
+    return not n % 2
```

Comment:
This can possibly be confusing because the function is called `is_even` but when you're reading the return, its saying `not n%2`. 

Not wrong by any means but had to take a double glance at what you were writing. 😆